tests/data/pinecone/Find me some data+faq-titles+<class 'filter'>+10.json

```diff
@@ -0,0 +1 @@
cument/d/1USv1ZaUhxC8d2r-iREknyjATzysb-g6DiAz5Banb-ro/edit?usp=drivesdk", "id": "i-2e62bca5e895879af470a81bd7b0ca98d4a15fee7484913aa098b01e15997c9a", "score": 0.202139601}, {"pageid": "8C7U", "status": "In progress", "title": "What are infohazards?", "url": "https://docs.google.com/document/d/1vBr_1iZOdHqRR_RQeLFe0V-7uUUe8_uyWOi1zSMKdQ8/edit?usp=drivesdk", "id": "i-0e24eea2c4c78a997ca1996c3a82880192b32576d259db1971e124e520de5e3d", "score": 0.184446961}, {"pageid": "6562", "status": "In progress", "title": "What external content would be useful to the Stampy project?", "url": "https://docs.google.com/document/d/1i2km30RwjrSV4vWjRRnbVKexdNg-m8dEg5RHstv5nOw/edit?usp=drivesdk", "id": "i-e2f6906a3ec935435272a6e6186a2276312c4051b63a3ee4b16e525b39da6f43", "score": 0.177998036}, {"pageid": "8266", "status": "In progress", "title": "What AGI safety reading lists are there?", "url": "https://docs.google.com/document/d/16NjnqKgVRbfKddACP04Q03VUgIkbAGg-GO_cla30IKg/edit?usp=drivesdk", "id": "i-66f8954110ee9dec4233f8b0a611aef57f960317fe5e2e5254c04502d8f00624", "score": 0.155111596}, {"pageid": "89ZT", "status": "In progress", "title": "How does Machine Learning work?", "url": "https://docs.google.com/document/d/1Ovn9t-am0iwVwXcoazd_MUhj0MGetfdq-wLw2a7bqbk/edit?usp=drivesdk", "id": "i-e79456204f40e64ec23a497d3299b3f6d5ed9ce42c552f8a4d40a65ced031a70", "score": 0.139714554}, {"pageid": "7754", "status": "In progress", "title": "What are some helpful AI policy resources?", "url": "https://docs.google.com/document/d/1PWQOYFjoZKgcolQ_1c0YrMxP8m0W0ioCsWrts1TtoCQ/edit?usp=drivesdk", "id": "i-aad565b931cf2449506782c2e6e0e4e253b6347d741456c208b76e3ef89fc44b", "score": 0.123128064}, {"pageid": "88FP", "status": "In review", "title": "How can we solve the alignment problem?", "url": "https://docs.google.com/document/d/1gL6UFk-7VqLpWPnFpJoZH_rFRBGEnmVcVcxBQN9I0QQ/edit?usp=drivesdk", "id": "i-ca01ac3704f0a4e92665f335d8b32d5f0ff6d10a4aef5ec2e84f42062b3e04f2", "score": 0.118116051}, {"pageid": "7611", "status": "In progress", "title": "How might a real-world AI system that receives orders in natural language and does what you mean look?", "url": "https://docs.google.com/document/d/1DQ_klodsbzvDK3_srKnxMsoCxvCfB9bfwmfG0H745hc/edit?usp=drivesdk", "id": "i-38123bd4e5b1b2bef652657a5dd58b8cb89dd98d2e54f64216f7522b78341347", "score": 0.103999317}, {"pageid": "5649", "status": "In progress", "title": "Why does there seem to have been an explosion of activity in AI in recent years?", "url": "https://docs.google.com/document/d/1JiXzy_Fc9jXHQvnIHXvyz4vV9E0Dumhw9hKI8reTJ88/edit?usp=drivesdk", "id": "i-89b72fa8b1891e7660b5875faf87e62f7fa7c1d0b547356884fac697c845ce28", "score": 0.103248358}, {"pageid": "8D5Y", "status": "In progress", "title": "What are some helpful AI policy ideas?", "url": "https://docs.google.com/document/d/1-R7jrF8MAjscm9Dv7-EQaXTBwqyurN2JHz6HJ-CnvdA/edit?usp=drivesdk", "id": "i-51fd2859f1036cc670e60ab4da91e03332217d2e0bba28cd6c1d77fecfe1a9e5", "score": 0.0787671804}]
```

Comment:
these files are simply the result of calling Pinecone for the given parameters in the file name - this allows mocks to have a real data as possible

Reply:
Good point. Having a flag check for before deploying would be nice.